pkg/pillar/types/zedmanagertypes.go

```diff
@@ -97,8 +97,10 @@ type SnapshotConfig struct {
 type AppRuntimeType int
 
 const (
-	AppRuntimeTypeUnSpecified AppRuntimeType = iota // AppRuntimeTypeUnSpecified
-	AppRuntimeTypeDocker                            // AppRuntimeTypeDocker, for docker compose apps
+	// AppRuntimeTypeUnSpecified is the default value for AppRuntimeType, indicating an unspecified runtime type.
```

Comment:
would be nice to squash to get rid of extra commit

Reply:
Done.